### ISSUE TITLE
Raw Configuration Editor Local Import Path

### ISSUE THREAD

**Issue by KalebMatthews** [WithOneVisionTechnologies/omnihive-manager-vscode#1]
Validator fails when using a local import path in a development environment.

```
{
	"name": "<Package Name>",
	"type": "<Worker Type>",
	"package": "",
	"version": "",
	"importPath": "../../../src/custom/<custom worker>",
	"default": false,
	"enabled": true,
	"metadata": {}
},
```

**Comment by doug-shontz** [comment]
Should be fixed in 4.0.1.

**Comment by KalebMatthews** [comment]
Works as expected
